app/components/spin-button.js

```diff
@@ -117,6 +121,7 @@ export default Ember.Component.extend({
   },
 
   setEnabled: function(){
+    this.set('showInFlightAnimation', false);
```

Comment:
This is unnecessary. See setProperties below and rename this prop back to `inFlight`